docs/notifications/getting-started.md

```diff
@@ -0,0 +1,220 @@
+---
+id: getting-started
+title: Getting Started
+description: How to get started with the notifications and signals
+---
+
+The Backstage Notifications System provides a way for plugins and external services to send notifications to Backstage users.
+These notifications are displayed in the dedicated page of the Backstage frontend UI or by frontend plugins per specific scenarios.
+Additionally, plugins can implement processors to send notifications through external channels like email, Slack, or MS Teams.
```

Comment:
Are Slack and MS Teams support in place? Maybe we can be more generic here.

Reply:
Not yet, rephrasing.